app/src/main/java/com/nmakademija/nmaakademija/ArticleActivity.java

```diff
@@ -22,5 +26,20 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                 article.getDescription());
         ((TextView) findViewById(R.id.new_title)).setText(
                 article.getTitle());
+        ActionBar actionBar = getSupportActionBar();
+        actionBar.setDisplayHomeAsUpEnabled(true);
+        actionBar.setSubtitle(R.string.news);
+        Glide.with(this).load(article.getTitleImage()).into((ImageView) findViewById(R.id.new_image));
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case android.R.id.home:
```

Comment:
It should work without this. Just change `AndroidManifest` activity definition to:
```xml
        <activity
            android:name=".SettingsActivity"
            android:parentActivityName=".MainActivity">
            <meta-data
                android:name="android.support.PARENT_ACTIVITY"
                android:value=".MainActivity" />
        </activity>
```